packages/calcite-components/src/components/combobox/combobox.tsx

```diff
@@ -1259,6 +1263,7 @@ export class Combobox
       }
       this.updateItems();
       this.filterItems("");
```

Comment:
Can you use the new flag to open after filtering for the new value? Similar to [above](https://github.com/Esri/calcite-design-system/pull/9618/files#diff-9bc2dae4daf67ad95f668837f3083752245fbc22e0a58b1e5a043830e05af02cL1056-L1057)?

Reply:
I don't think so. That one will only open when there is text present in the input and this one should open regardless.

Reply:
Gotcha. Maybe the function argument can be renamed to distinguish this.

Reply:
Yup, will do. 👍